packages/web-react/src/components/RadioField/RadioField.tsx

```diff
@@ -1,14 +1,25 @@
 import classNames from 'classnames';
 import React from 'react';
-import { useStyleProps } from '../../hooks/styleProps';
+import { useDeprecationMessage, useStyleProps } from '../../hooks';
 import { SpiritRadioFieldProps } from '../../types';
 import { useRadioFieldStyleProps } from './useRadioFieldStyleProps';
 
 export const RadioField = (props: SpiritRadioFieldProps): JSX.Element => {
   const { classProps, props: modifiedProps } = useRadioFieldStyleProps(props);
-  const { id, label, value, isDisabled, isChecked, onChange, ...restProps } = modifiedProps;
+  const { id, label, helperText, value, isDisabled, isChecked, onChange, ...restProps } = modifiedProps;
   const { styleProps, props: otherProps } = useStyleProps(restProps);
 
+  useDeprecationMessage({
```

Comment:
I was also thinking, but unfortunately it uses the same dictionary where `error` is deprecated, so it should therefore be in that component if it offers it in the props.
Otherwise, we could additionally throw it directly in RadioProps, but that seems unnecessary to me personally. Like when it uses the same dictionary for similar components.